### ISSUE TITLE
use .ssh/config

### ISSUE THREAD

**Issue by jots** [bastienleonard/hutte#1]
Can hutte be made to use ~/.ssh/config so can specify name that is a "Host" in .ssh/config and it will use the full hostname, port, identityfile etc?

**Comment by bastienleonard** [comment]
Yep, looks like it. The SSH library used in Hutte reads ~/.ssh/config by default. It only parses some part of the options though, see https://net-ssh.github.io/ssh/v2/api/classes/Net/SSH/Config.html.

I just made a quick test and could get it to to work, you just need to pass the “Host” value in place of the hostname.
However, if you want the username to be deduced from the config file, it needs to be nil (`''` won't work).

(I tested using the master branch. Unless you have some old code that might break because of API changes, I don't recommend using the stable branch currenty, it's horribly outdated.)
